admin-auth/src/main/kotlin/com/epam/drill/admin/auth/exception/NotAuthenticatedException.kt

```diff
@@ -15,5 +15,4 @@
  */
 package com.epam.drill.admin.auth.exception
 
-class IncorrectCredentialsException : Exception("Username or password is incorrect") {
-}
\ No newline at end of file
+class NotAuthenticatedException(message: String? = null): Exception(message ?: "User is not authenticated")
```

Comment:
Swap for "Invalid credentials" or "Authentication required"
"User is not authenticated" sounds like we're talking about somebody in 3rd person
-> It will look weird on UI